### PR TITLE
Backend identifying info

### PR DESCRIPTION
**Pull request by floresj**
**Notes**
- Updated field validation and added endpoints for eye color, hair color, sex, weight and height
- Updated api documentation

### CODE REVIEW

File: api/model/form/weight_field.go [hunk 1]
```diff
@@ -5,9 +5,10 @@ import "fmt"
 // WeightField stores a persons weight
 type WeightField int64
 
+// Valid validates that a person weight is within the specified range
 func (f WeightField) Valid() (bool, error) {
-	if f < 0 || f > 2000 {
-		return false, fmt.Errorf("Invalid weight")
+	if f < 10 || f > 999 {
```
Comment (adelevie) [inline api/model/form/weight_field.go]: @billy22g let's ask about some of these height and weight validations. These are the limits on the current e-QIP  form, but maybe worth changing?

File: api/model/form/weight_field_test.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+func TestWeightField(t *testing.T) {
+	tests := []struct {
+		Field    WeightField
+		Expected bool
+	}{
+		{10, true},
+		{0, false},
+		{1000, false},
+		{999, true},
+		{10, true},
```
Comment (adelevie) [inline api/model/form/weight_field_test.go]: Get rid of this later, not blocking merge though.

File: api/model/form/weight_field.go (hunk 1 above)
Comment (billy22g) [inline api/model/form/weight_field.go]: @adelevie Sounds good to me. Do you want to be more stringent? I think being pretty flexible is better than too stringent, which is why I'm not particularly offended by the current validation criteria.

**Comment by adelevie** [comment]
I want to be less stringent.

**Comment by billy22g** [comment]
I actually think it's a pretty smart validation. The heaviest person ever is still in the 3 digits, and an adult of any size is never going to be in the single digits. So if it's a single digit or more than 3 digits, it's clearly an error. Right?

**Pull request merged**